src/components/common/ColonyActions/ActionsListItem/ActionsListItem.tsx

```diff
@@ -88,7 +88,7 @@ const ActionsListItem = ({
   const navigate = useNavigate();
 
   const handleActionRedirect = () =>
-    navigate(`/colony/${colony?.name}/tx/${transactionHash}`);
+    navigate(`/${colony?.name}?tx=${transactionHash}`);
```

Comment:
I guess this is a question more for @arrenv 

Why was the query format chosen over a path?

Reply:
It is not something I specifically spec'd, the way I understand it is that this was simply the easiest way to make transactions work in the side panel, no matter what page it was shared from.